src/adapter/aws-lambda/index.ts

```diff
@@ -1,4 +1,4 @@
 // @denoify-ignore
-export { handle, streamHandle } from './handler'
+export { handle, streamHandle, type APIGatewayProxyResult } from './handler'
```

Comment:
Could you avoid writing `type` inside `{}`? We use the syntax.

```ts
export { handle, streamHandle } from './handler'
export type { APIGatewayProxyEvent } from './handler'
```

Reply:
Sure, done